src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/ConnectionDetails.cs

```diff
@@ -592,31 +592,41 @@ public string AzureAccountToken
         }
 
         public bool IsComparableTo(ConnectionDetails other)
```

Comment:
Since it's now comparing all properties we should probably just make this `public override bool Equals(object? obj)`

Reply:
I thought about that, but we are still not covering 'GroupId' in this comparison, as it's managed by ADS layer and didn't seem to make a difference when editing a connection from STS layer.

Reply:
Gotcha, makes sense